adapter/ffmpeg_frame.php

```diff
@@ -34,11 +34,11 @@ public function getPresentationTimestamp() {
     }
     
     public function resize($width, $height, $cropTop = 0, $cropBottom = 0, $cropLeft = 0, $cropRight = 0) {        
-        return $this->adaptee->resize($width, $height, $cropTop, $cropBottom, $cropLeft, $cropRight);
+        $this->adaptee->resize($width, $height, $cropTop, $cropBottom, $cropLeft, $cropRight);
```

Comment:
FFmpegFrame::resize is a void function, we're returning the result of a void function.